lib/hca/configuration.rb

```diff
@@ -34,20 +17,8 @@ def service_name
       'HCA'
     end
 
-    def ssl_options
-      ssl = {
-        verify: true
```

Comment:
This option is unnecessary because the [default value is `true`](https://github.com/lostisland/faraday/blob/d77c9efee9b12763f685f46df470eb22351154f5/lib/faraday/adapter/net_http.rb#L210)